src/Analyzers/Core/Analyzers/SimplifyInterpolation/Helpers.cs

```diff
@@ -72,11 +72,11 @@ private static IOperation Unwrap(IOperation expression)
             if (expression is IInvocationOperation { TargetMethod: { Name: nameof(ToString) } } invocation &&
                 HasNonImplicitInstance(invocation) &&
                 !syntaxFacts.IsBaseExpression(invocation.Instance.Syntax) &&
-                !invocation.Instance.Type.IsRefLikeType)
+                !invocation.Instance.Type!.IsRefLikeType)
```

Comment:
What test above is giving us confidence that `Instance.Type` is non-null here?

Reply:
The fact that `null` can never be an instance for an invocation, and therefore the type will never be `null`.

Reply:
Hmm, wish we had a way of annotating such a relationship here but we don't :(